network/p2p/topic_validator_test.go

```diff
@@ -60,9 +57,12 @@ func TestTopicValidator_Unstaked(t *testing.T) {
 		return nil
 	}
 
+	pInfo2, err := p2p.PeerAddressInfo(identity2)
+	require.NoError(t, err)
+
 	// node1 is connected to node2
 	// sn1 <-> sn2
-	require.NoError(t, sn1.AddPeer(context.TODO(), *host.InfoFromHost(sn2.Host())))
+	require.NoError(t, sn1.AddPeer(context.TODO(), pInfo2))
```

Comment:
so the fix was to use `p2p.PeerAddressInfo(identity2)` instead of `host.InfoFromHost(sn2.Host())` to get the peer's `PeerInfo`?

Reply:
Updated the description.